datasrc/network.py

```diff
@@ -374,6 +374,10 @@
 		NetIntAny("m_Layer"),
 		NetIntAny("m_EntityClass"),
 	]),
+
+	NetEventEx("MapSoundWorld:Common", "map-sound-world@netevent.ddnet.org", [
+		NetIntAny("m_SoundId", 0),
```

Comment:
Was the 0 intentional? If not:
```suggestion
		NetIntAny("m_SoundId"),
```